modules/300-prometheus/templates/grafana/service.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
   annotations:
     prometheus.deckhouse.io/port: "8443"
     prometheus.deckhouse.io/tls: "true"
+    prometheus.deckhouse.io/sample-limit: "5000"
```

Comment:
This is not the only place. We have another monitor for Grafana right here
https://github.com/deckhouse/deckhouse/blob/main/ee/fe/modules/340-monitoring-applications/templates/service-monitors/grafana.yaml

Reply:
Changed it as well!